### PR TITLE
feat: avoid expired thumbnails

### PR DESCRIPTION
**Pull request by jemayn**

### CODE REVIEW

File: src/Limbo.Integrations.Skyfish/Http/SkyfishHttpClient.cs [hunk 1]
```diff
@@ -113,6 +117,12 @@ public class SkyfishHttpClient : HttpClient {
             return JsonUtils.ParseJsonObject(response.Body, SkyfishVideo.Parse);
         }
 
+        private string GetVideoThumbnail(int videoId) {
+            var response = Get($"/search?unique_media_id={videoId}&return_values=unique_media_id+height+width+title+description+thumbnail_url+thumbnail_url_ssl+filename+file_disksize+file_mimetype");
+            SkyfishVideo video = JsonUtils.ParseJsonObject(response.Body, SkyfishVideo.Parse);
+            return video.ThumbnailUrlSsl;
+        }
+
```
Comment (abjerner) [inline src/Limbo.Integrations.Skyfish/Http/SkyfishHttpClient.cs:125]: Why the private method? Couldn't the method body just be part of the `GetThumbnailUrl`?

File: src/Limbo.Integrations.Skyfish/Limbo.Integrations.Skyfish.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.0.0-beta001</Version>
+    <Version>1.0.0-beta002</Version>
```
Comment (abjerner) [inline src/Limbo.Integrations.Skyfish/Limbo.Integrations.Skyfish.csproj:9]: Ideally don't change this. I prefer bumping the version number when ready to push a new release. The change is then done in a separate commit without any actual code changes - which for instance is nice if we later have to revert some code changes.

**Comment by abjerner** [comment]
Hi @jemayn 

I guess this one depends how you're seeing our integration packages. At least for our other integration packages, the idea is that they follow the API endpoints/methods as close as possible.

So in this case, couldn't we skip updating the integration package, and then have the Umbraco package call `client.GetViideo(123).ThumbnailUrlSsl` instead? That would avoid a bit of redundant code in the integration package.

**Comment by jemayn** [comment]
Closing as we're moving on with #4 instead.

**Pull request closed without merging**